src/Riok.Mapperly/AnalyzerReleases.Shipped.md

```diff
@@ -79,6 +79,9 @@ RMG030  | Mapper   | Error    | Reference loop detected while mapping to an init
 RMG031  | Mapper   | Warning  | Reference loop detected while mapping to a constructor member
 RMG032  | Mapper   | Warning  | The enum mapping strategy ByName cannot be used in projection mappings
 RMG033  | Mapper   | Info     | Object mapped to another object without deep clone
+RMG034  | Mapper   | Error    | Derived source type is duplicated
```

Comment:
"Duplicated" could lead to confusion. Maybe
> Derived source type is specified multiple times. You may only specify it once.

Reply:
Adjusted.